manifests/00-crd-extension-console-pod-log-link.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: consolepodloglinks.console.openshift.io
+  annotations:
+    displayName: ConsolePodLogLinks
+    description: Extension for customizing OpenShift web console pod log links
+spec:
+  scope: Cluster
+  group: console.openshift.io
+  versions:
+  - name: v1
+    served: true
+    storage: true
+  names:
+    plural: consolepodloglinks
+    singular: consolepodloglink
+    kind: ConsolePodLogLink
+    listKind: ConsolePodLogLinkList
+  additionalPrinterColumns:
+  - name: Text
+    type: string
+    JSONPath: .spec.text
+  - name: HrefTemplate
+    type: string
+    JSONPath: .spec.hrefTemplate
+  - name: Age
+    type: date
+    JSONPath: .metadata.creationTimestamp
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          type: object
+          description: Represents console pod log link customizations spec
+          required:
+          - text
+          - hrefTemplate
+          properties:
+            text:
+              type: string
+              description: Text of the link
+            hrefTemplate:
+              type: string
+              description: Absolute secure URL including pod name or uid, container name, namespace, and labels variables for the pod log link (must use https).  e.g., https://example.com/logs?name=\${name}&container=\${container}&namespace=\${namespace}&labels=\${labels}
```

Comment:
We'll want to document specifically all the replacements we support and the exact format